app/controllers/concerns/api.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  extend ActiveSupport::Concern
+
+  def ok(status: 200, content: nil, root: nil)
+    if content.blank?
+      render status: status, nothing: true
+    else
+      respond_to do |format|
+        format.json { render status: status, json: content.as_json }
+        format.xml  { render status: status, xml: content.to_xml(root: root, skip_types: true) }
+        format.yaml { render status: status, text: content.to_yaml, content_type: "text/yaml" }
```

Comment:
do we have a specific use case for yml and xml? otherwise i'd be in favor of keeping it thin, meaning json only

Reply:
Not really, I can remove them yeah... Updating